test/regression/cases/ddot_metrics/datadog-agent/datadog.yaml

```diff
@@ -17,3 +17,16 @@ process_config:
 telemetry:
   enabled: true
   checks: '*'
+
+otelcollector:
+  converter:
+    enabled: true
+  enabled: true
+  extension_timeout: 0
+  flare:
+    timeout: 60
+
+agent_ipc:
```

Comment:
I'm unfamiliar with this config, why is it needed? Should it be on more experiments?